docs/code-search/features.mdx

```diff
@@ -95,6 +95,29 @@ You get the following improvements:
 - **Reworked fuzzy finder ⌘K**: Find files and symbols quickly and easily with our whole new fuzzy finder
 - **File actions**: Like open in editor and open on code host
 
+## Personalized search ranking
+
+<Callout type="info">The personalized search ranking experience is available for Enterprise users only.</Callout>
+
+Sourcegraph Enterprise users can get more personalized and better-ranked search results in the search bar for their codebases. With this experience enabled, you can get intelligently ranked results for your full query for keyword searches like "data router". You also get results boosted from repositories you’ve recently contributed to, making it easier to find the code you care about faster.
```

Comment:
I would remove the sentence "With this experience enabled, you can get intelligently ranked results for your full query for keyword searches like "data router"." There are actually two separate features shipping in 6.2:
* Improved ranking for keyword searches like "data router". This is always on, and cannot be configured. It is pretty "behind the scenes" and users really shouldn't need to worry about it.
* Personalized ranking, specifically boosted results from repos you recently contributed to. This is controlled by the setting `experimentalFeatures.boostRelevantRepositories`.